src/homeWork9/ArrayValueCalculator.java

```diff
@@ -0,0 +1,32 @@
+package homeWork9;
+
+public class ArrayValueCalculator {
+
+    public int doCalc(String[][] array) {
+
+        int result = 0;
+        int rowNumber = 0;
+        int columnNumber = 0;
+
+        if (array.length != 4){
+            throw new ArraySizeException("The array does not have 4 rows");
+        }
+
+        for (String[] row : array) {
+            rowNumber++;
+            if (row.length != 4) {
+                throw new ArraySizeException("In row: " + rowNumber + " the number of elements is not 4");
+            }
+            for (String column : row) {
+                columnNumber++;
+                try {
+                    result = result + Integer.parseInt(column);
+                } catch (Exception e) {
```

Comment:
The method `'Integer.parseInt()` throws `NumberFormatException`. Replace `Exception` with `NumberFormatException`.